packages/flutter/lib/src/rendering/table.dart

```diff
@@ -607,9 +607,57 @@ class RenderTable extends RenderBox {
   void describeSemanticsConfiguration(SemanticsConfiguration config) {
     super.describeSemanticsConfiguration(config);
     config.role = SemanticsRole.table;
+    config.isSemanticBoundary = true;
     config.explicitChildNodes = true;
   }
 
+  /// Overrides this method to create semantics nodes for rows between table and table cells.
+  @override
+  void assembleSemanticsNode(
+    SemanticsNode node,
+    SemanticsConfiguration config,
+    Iterable<SemanticsNode> children,
+  ) {
+    assert(_children.length == _rows * _columns);
+    final List<SemanticsNode> rows = <SemanticsNode>[];
+
+    for (int i = 0; i < _rows; i++) {
+      final Rect rowBox = getRowBox(i);
+      // Skip row if it's empty
+      if (rowBox.height == 0) {
+        continue;
+      }
+      final SemanticsNode newRow = SemanticsNode(
+        showOnScreen: () {
+          showOnScreen(descendant: this, rect: rowBox);
+        },
+      );
+
+      final SemanticsConfiguration configuration =
+          SemanticsConfiguration()
+            ..indexInParent = i
```

Comment:
is this going to be used by the row index? if so, does the cell also uses this to set the col index? we should unify the way to set these indices in web.

Reply:
updated indexInParent for cells